org.eclipse.lsp4e/src/org/eclipse/lsp4e/ui/LSPImages.java

```diff
@@ -34,219 +34,222 @@
 
 public final class LSPImages {
 
-	private LSPImages() {
```

Comment:
@gerdleon , it looks like the line tabs are now spaces, could you convert them back to tabs?

Reply:
> I'm also not sure deriving null icon from the class one makes a lot of sense: null is not about defining something, but rather an undefined reference, or a reference to nothing, IMO, the simplest would be to put the symbol on a transparent background, similarly to number or string or boolean icons which are about "native" concepts.

Makes sense.